### PR TITLE
Feat: Render the sitemaps (extension & webapp).

### PR DESCRIPTION
**Pull request by Jamerrone**

### CODE REVIEW

File: projects/@shared/components/tree-menu.vue [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    elements: {
+      type: Array,
+      default: () => [],
+    },
+    text: {
+      type: String,
+      default: '',
+    },
+  },
+  data() {
```
Comment (vvandijck) [inline projects/@shared/components/tree-menu.vue]: This is no longer used.

File: projects/web-app/src/views/sitemap.view.vue [hunk 2]
```diff
@@ -1,37 +1,54 @@
...
+  </div>
 </template>
 
 <script>
 import { computed } from 'vue';
-import useHead from '@/composables/use-head';
-import PanelSection from '@shared/components/panel-section';
 import ExternalLink from '@shared/components/external-link';
+import PanelSection from '@shared/components/panel-section';
+import TreeMenu from '@shared/components/tree-menu';
```
Comment (vvandijck) [inline projects/web-app/src/views/sitemap.view.vue:50]: This is the only line that is changed right?

File: projects/web-app/src/views/sitemap.view.vue [hunk 3]
```diff
@@ -1,37 +1,54 @@
...
+        v-else
+        v-for="sitemap in sitemaps"
+        :key="Object.keys(sitemap)[0]"
+        class="sitemap-view__sitemap"
+      >
+        <p class="sitemap-view__sitemap-url">{{ Object.keys(sitemap)[0] }}</p>
+        <tree-menu
+          v-if="Object.values(sitemap)[0].elements[0].elements"
+          :name="Object.values(sitemap)[0].elements[0].name"
+          :elements="Object.values(sitemap)[0].elements[0].elements"
```
Comment (vvandijck) [inline projects/web-app/src/views/sitemap.view.vue]: Can't these be methods?

File: projects/browser-extension/vue-app/src/views/sitemap.view.vue [hunk 4]
```diff
@@ -5,9 +5,23 @@
...
+        v-else
+        v-for="sitemap in sitemaps"
+        :key="Object.keys(sitemap)[0]"
+        class="sitemap-view__sitemap"
+      >
+        <p class="sitemap-view__sitemap-url">{{ Object.keys(sitemap)[0] }}</p>
+        <tree-menu
+          v-if="Object.values(sitemap)[0].elements[0].elements"
+          :name="Object.values(sitemap)[0].elements[0].name"
+          :elements="Object.values(sitemap)[0].elements[0].elements"
```
Comment (vvandijck) [inline projects/browser-extension/vue-app/src/views/sitemap.view.vue]: Can't these be methods?

File: projects/@shared/components/tree-menu.vue (hunk 1 above)
Comment (Jamerrone) [reply projects/@shared/components/tree-menu.vue]: Good catch, I forgot that.

Comment (Jamerrone) [reply projects/@shared/components/tree-menu.vue:38]: Good catch, I forgot that.

File: projects/web-app/src/views/sitemap.view.vue (hunk 2 above)
Comment (Jamerrone) [reply projects/web-app/src/views/sitemap.view.vue:50]: Yup. `import TreeMenu from '@shared/components/tree-menu';` was added, nothing more.

**Review by vvandijck — APPROVED** [review]

**Pull request merged**